index.html

```diff
@@ -13,7 +13,21 @@
 </head>
 
 <body>
+  <footer>
+    <div class="botoes">
 
+    </div>
+
+    <div class="componenteTempo">
```

Comment:
Vamos patronizar nomes de classes separados por traços, no css
```suggestion
    <div class="componente-tempo">
```